modules/odc-accessing-developer-perspective.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// web-console/odc-about-developer-perspective.adoc
+
+[id="accessing-developer-perspective_{context}"]
+= Accessing Developer Perspective
+
+The *Developer* perspective in the {product-title} web console provides work flows specific to developer use cases.
+
+You can access the *Developer* perspective from the web console as follows:
+
+.Prerequisites
+
+* Ensure that you have link:https://docs.openshift.com/container-platform/latest/web-console/web-console.html[logged in to the web console].
```

Comment:
for internal references, we must use this format for links:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references